tools/macos/README.md

```diff
@@ -51,7 +51,7 @@ brew install scala
 # install pip
 sudo easy_install pip
 # install script prerequisites
-sudo -H pip install docker==2.2.1 ansible==2.5.2 jinja2==2.9.6 couchdb==1.1 httplib2==0.9.2 requests==2.10.0' | bash
+sudo -H pip install docker==2.2.1 ansible==2.5.2 jinja2==2.9.6 couchdb==1.1 httplib2==0.9.2 requests==2.10.0 | bash
```

Comment:
this `'` matches the one on line 40 https://github.com/apache/incubator-openwhisk/pull/3657/files#diff-35b51f3cf01d8ad81f668514d8c74622R40. It's not an accident.